src/lib/Timeline.scss

```diff
@@ -195,7 +195,7 @@ $weekend: rgba(250, 246, 225, 0.5);
     .rct-hl-even,
     .rct-hl-odd {
       border-bottom: $border-width solid $border-color;
-      box-sizing: content-box;
```

Comment:
any particular reason why this is changed from `content-box` to `border-box`?

Reply:
@gaston-niglia oh okay I read more on this and now I understand. This is brilliant. Thank you!